src/Goerli-DssSpell.t.sol

```diff
@@ -404,6 +476,40 @@ contract DssSpellTest is GoerliDssSpellTestBase {
         assertEq(OsmAbstract(addr.addr("PIP_TOKEN")).bud(READER), 1);
     }
 
+    function testPSMs() public {
+
+        vote(address(spell));
+        scheduleWaitAndCast(address(spell));
+        assertTrue(spell.done());
+
+        bytes32 _ilk = "PSM-PAX-A";
+        checkPsmIlkIntegration(
+            _ilk,
+            GemJoinAbstract(reg.join(_ilk)),
+            ClipAbstract(reg.xlip(_ilk)),
+            reg.pip(_ilk),
+            PsmAbstract(chainLog.getAddress("MCD_PSM_PAX_A")),
```

Comment:
Normally we fetch addresses from `addr.addr("XXX")` for tests.

Reply:
fixed